compiler/circle-opselector-test/test.lst

```diff
@@ -0,0 +1,7 @@
+Part_While_001
```

Comment:
`while` doesn't support?

Reply:
No, `while` doesn't support. `Part_While_001` model can't be used for test but just added in test list